Cargo.toml

```diff
@@ -1,19 +1,24 @@
 [package]
 name = "didcomm-rs"
-version = "0.6.7"
+version = "0.6.6"
 authors = ["Ivan Temchenko <35359595i@gmail.com>"]
```

Comment:
also add yourselves into authors if you like =)

Reply:
adjusted :)